### ISSUE TITLE
Do it works on node 19.6.0?

### ISSUE THREAD

**Issue by crystalfp** [balena-io-modules/drivelist#423]
Upgraded node from 18.13.4 to 19.6.0 and now npm update complains that `Unsupported engine` because it requires ` node: '>=16 < 19'`.
Later today I will check if it still works, but in the meantime could a new version that works on node 19 be released?
Thanks!
mario

**Comment by crystalfp** [comment]
Strange. My installed version is 11.1.0 but the site pointed by npm said 6.4.3. I'm working on Windows 11.
Thanks for clarifying!
mario

**Comment by lurch** [comment]
The comments in #420 suggest that it only builds reliably on Node 14? :shrug:

**Comment by lurch** [comment]
> My installed version is 11.1.0 but the site pointed by npm said 6.4.3.

Yeah, https://github.com/balena-io-modules/drivelist/tags says the latest "tag" is v11.1.0 but https://github.com/balena-io-modules/drivelist/releases says the latest "release" is v6.4.3 :confused:

**Comment by crystalfp** [comment]
It works under Node 19.8.1 so it seems a small change to package configuration is sufficient

**Comment by bfan1256** [comment]
Any updates to this?

**Comment by pectum83** [comment]
I have troubles using it with node 20.9.0
`error: 'Uncaught exception:  Could not locate the bindings file. Tried:\n' +
  ' → myProject/node_modules/.pnpm/drivelist@12.0.2/node_modules/drivelist/build/drivelist.node\n' `

Does anyone use it with node 20 successfully ?
